vendor/github.com/flosch/pongo2/lexer.go

```diff
@@ -70,7 +70,7 @@ type lexer struct {
 func (t *Token) String() string {
 	val := t.Val
 	if len(val) > 1000 {
-		val = fmt.Sprintf("%s...%s", val[:10], val[len(val)-5:len(val)])
```

Comment:
@przemyslaw-dobrowolski-cl the short answer is for convenience, you can format code in fire and forget mode. Otherwise you have to always revert `vendor` or do some sort of `grep -v vendor`. The long answer is that I'd recommend you go and vote fort https://github.com/golang/go/issues/19090 then the issue will be solved.